### DISCUSSION TITLE
[Bug] Update failed. becasue of git conflict. how can i solve it

### DISCUSSION THREAD

**Discussion by mnnna** [ScoopInstaller/Scoop#5890] [Help and Questions]
#### Current Behavior

<img width="744" alt="企业微信截图_17063241868915" src="https://github.com/ScoopInstaller/Scoop/assets/100188748/e7d83859-e37f-49a1-b330-076abbc7587d">

### System details

**Windows version:** [e.g. 7, 8, 10, 11]
## Windows 11
**OS architecture:** [e.g. 32bit, 64bit, arm64]
## 64bit
**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
## 5.1
**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
//# Your configuration here
```

**Comment by rashil2000** [comment]
Can you cd into the Scoop directory (`~\scoop\apps\scoop\current`) and check `git status` there?
